CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 4.1.12
+
+* Rename cotsoc_noncontrib to cotisations_non_contributives
```

Comment:
You should also do that with cotsoc_contrib etc.
Look for all these badly named cotosoc_* ;-)